src/nvhttp.cpp

```diff
@@ -1148,8 +1196,48 @@ namespace nvhttp {
    */
   void
   erase_all_clients() {
```

Comment:
Maybe we should do a force close as part of the unpair all?

Reply:
Makes sense. "Unpair All" now force closes the open app/session. I've also updated the language to specify that the user only might need to do it themselves if unpairing individual devices.

Reply:
Sorry, I am not seeing the change that would do the force close. Can you direct me to that part of the code?

Reply:
Sure: https://github.com/LizardByte/Sunshine/pull/2042/commits/1b6a828b1cdcda2854097b2d5788ee1d48f9dff9